examples/banananation/src/main/java/gov/nasa/jpl/aerie/banananation/CustomValueMappers.java

```diff
@@ -0,0 +1,8 @@
+package gov.nasa.jpl.aerie.banananation;
+
+import gov.nasa.jpl.aerie.banananation.activities.Banannotation;
+import gov.nasa.jpl.aerie.merlin.framework.annotations.MissionModel;
+
+@MissionModel.WithMetadata(name="label", annotation= Banannotation.class)
+public class CustomValueMappers {
+}
```

Comment:
We pair-programmed moving WithMetadata to package-info.java

The implication is that mission models will need to include `@WithMetadata` in their own `package-info.java` file, since we cannot provide that out-of-the-box. We should add it to the aerie-mission-model-template repo so that new users don't need to think about this.